ansible/roles/ocp4-workload-debugging-workshop/tasks/post_workload.yml

```diff
@@ -145,34 +181,33 @@
     data:
       msg: |
         Username: {{ item }}
-        Password: {{ workshop_openshift_user_password }}
-        Workshop User Guide: http://bookbag-{{ item | regex_replace('^user(.*)$', '\1') }}-{{ workshop_operator_namespace }}.{{ route_subdomain }}/workshop
+        Password: {{ workshop_openshift.user_password }}
+        Workshop User Guide: http://bookbag-{{ item | regex_replace('^user(.*)$', '\1') }}-workshop-infra.{{ route_subdomain }}/workshop
   loop: "{{ users }}"
 
 - name: output workshop info
   agnosticd_user_info:
     msg:
-      - ""
-      - "Debugging Workshop provisioned for {{ user_count }} user(s)"
-      - ""
-      - "Workshop User Guide: http://bookbag-XX-{{ workshop_operator_namespace }}.{{ route_subdomain }}/workshop"
-      - "Etherpad: http://etherpad-{{ workshop_operator_namespace }}.{{ route_subdomain }}"
-      - ""
-      - "You should share the Etherpad URL (or a shortlink for it) with attendees to self-assign a user."
-      - "It is all they will need to get started!"
-      - ""
-      - "OpenShift credentials for attendees: '{{ workshop_openshift_user_name }}' / '{{ workshop_openshift_user_password }}'"
-      - ""
-      - "OpenShift Console: {{ console_url }}"
-      - "Cluster admin login with '{{ ocp_username }}' / '{{ workshop_openshift_admin_password }}'"
-      - ""
-      - "OpenShift/Kubernetes API (use with oc login): {{ master_url }}"
-      - ""
-      - "CodeReady Console: http://codeready-codeready.{{ route_subdomain }}"
-      - "Admin login with 'admin' / 'admin'"
-      - ""
-      - "NOTE: Workspaces in CodeReady are provisioned asynchronously and may not"
-      - "be accessible until rollout finishes shortly."
+      - "user.info: "
```

Comment:
I actually think you don't need to prefix with user.info when using the agnosticd_user_info module.